plugins/countdown/test_countdown_row_save.py

```
@@ -0,0 +1,122 @@
+    core = os.environ.get("LEDMATRIX_CORE", "")
+    for candidate in (core, str(PLUGIN_DIR.parents[2] / "LEDMatrix")):
+        if candidate and (Path(candidate) / "src" / "plugin_system").is_dir():
+            sys.path.insert(0, candidate)
+            break
+    sys.path.insert(0, str(PLUGIN_DIR))
+    try:
+        spec = importlib.util.spec_from_file_location(
+            "countdown_manager", PLUGIN_DIR / "manager.py")
+        module = importlib.util.module_from_spec(spec)
+        spec.loader.exec_module(module)
+    except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)